### ISSUE TITLE
Feature Request: Move Photos to New Person

### ISSUE THREAD

**Issue by jtrees** [pulsejet/memories#336]
In the People view it is currently possible to move wrongly tagged photos from one person to another. This only works as long as the target person "already exists" though (i.e. there are already other photos tagged with this person).

It would be great to have the ability to move a wrongly tagged image to new person that isn't already in the database.

**Comment by luxzg** [comment]
I see this as great addition to #475 and actually I came here to see if these two requests already existed.

As a workaround, we could maybe just get ability to create a new (blank) person.

In a way, splitting photos to new person is basically create new person + move photos to it.

Btw, as a temporary workaround for myself, does anyone know if we are allowed to add new person in database directly? I've modified table `oc_recognize_face_clusters` to change title of a person (due to bug that's currently deleting person when being renamed in web UI), and it worked great. I could manually add few people to my database, but I'm uncertain if it would break something else, or require additional changes somewhere else.

Other workaround I was thinking about is finding a single-photo cluster of someone unimportant, adding photos there, renaming to my new persom, and excluding that 1 original photo. But, again, uncertain if that would have some leftovers from the original person somehow... possibly later getting that person's photos again in there.

If anyone has any ideas, or additional info, I'd be happy to hear them. Thanks!
